### ISSUE TITLE
tests/mock_vws/test_query.py::TestMaximumImageFileSize::test_png[Real Vuforia] FAILED

### ISSUE THREAD

**Issue by adamtheturtle** [VWS-Python/vws-python-mock#837]
See https://github.com/VWS-Python/vws-python-mock/runs/1628674699?check_suite_focus=true

```
        with pytest.raises(requests.exceptions.ConnectionError):
82
>           query(
83
                vuforia_database=vuforia_database,
84
                body=body,
85
            )
86
E           Failed: DID NOT RAISE <class 'requests.exceptions.ConnectionError'>
```

**Comment by adamtheturtle** [comment]
On GitHub actions we get `413 Request Entity Too Large` with the following content:

```
'<html>\r\n<head><title>413 Request Entity Too Large</title></head>\r\n<body bgcolor="white">\r\n<center><h1>413 Request Entity Too Large</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n'
```

I am yet to figure out what is different between CI and my local machine where everything passes.

**Comment by adamtheturtle** [comment]
The test fails on my Mac in an Ubuntu container.

**Comment by adamtheturtle** [comment]
Next step: Try the request without `requests`.

**Comment by adamtheturtle** [comment]
Next step also: Create a minimal reproducible script which displays different behaviour on macOS vs Linux.

**Comment by adamtheturtle** [comment]
SOLVED!

The original test was really just testing a bug in `urllib3` which was fixed in version 1.26.0 with https://github.com/urllib3/urllib3/pull/1524.
